src/shader.h

```diff
@@ -10,6 +10,56 @@
 
 namespace bgfx
 {
+	struct DescriptorType
+	{
+		enum Enum
+		{
+			Undefined,
```

Comment:
In the rest of bgfx `Count` is used as undefined or don't care. Is there any specific reason to have `undefined`?

Reply:
Probably not. My thought process was to have a known ID for 'no descriptor type' but that should already be obvious from the uniform type and then this data is discarded. I'll remove it.